docs/articles/documentation/guides/advanced-guides/speed-up-test-execution.md

```diff
@@ -103,3 +104,37 @@ This approach ensures that the page model object is created only once per test r
 > Important! If you export the page model class and create an instance in every test file, JavaScript creates a new page object per test file. This increases test execution time and memory consumption.
 
 For more on page models, read [Page Model](../../guides/concepts/page-model.md).
+
+## Enable Server-Side Caching
+
+> Important! Server-side caching is **experimental**.
+>
+> Don't use caching if you run into compatibility issues with your tests.
+
+TestCafe can store webpage assets (like styles, scripts, and images) in its server cache. When TestCafe visits the page again, it pulls assets from the cache instead of requesting them from server.
+
+You can enable caching in one of the following ways:
+
+* [--cache](../../reference/command-line-interface.md#cache) CLI option
+
+  ```sh
+  testcafe chrome my-tests/ --cache
+  ```
+
+* [cache]() configuration file property
+
+  ```json
+  {
+  "cache": true
+  }
+  ```
+
+* [cache]() API option
+
+  ```js
+  runner.run({ cache: true });
+  ```
+
+If the tested application loads assets that are cached by the browser, enable server-side caching to decrease test run time.
```

Comment:
lots of issues with this sentence. it doesn't do a good job of explaining the rationale behind the feature. did I get it right?
```suggestion
Enable server-side caching to decrease test run time, particularly when you expect your users to rely on browser caching.
```

Reply:
Users can't rely on caching, the app can

```suggestion
Enable server-side caching to decrease test run time, particularly when your application relies on browser caching.
```